structuredheadings/plugin.js

```diff
@@ -301,22 +317,23 @@
       setupElements(editor);
       setupCommands(editor);
       setupStyles(editor);
+      self.currentScheme = editor.config.autonumberDefaultScheme;
 
       editor.on("dataReady", function () {
         self.currentScheme = self.detectScheme();
       });
 
       //Format Dropdown
       editor.ui.addRichCombo("NumFormats", {
-        label: "Formats",
-        title: "Numbering Formats",
+        label: "Formatting",
+        title: "Text Formatting",
```

Comment:
The title gets added only as a title attribute, so it doesn't actually display anywhere except for mouseover (I think it should be consistent with the aria label as well, though my accessibility knowledge is fuzzy and lacking in this area)  

The Formatting part is to handle the default state (only really shows when you are in a non-paragraph/h block, such as a new table cell)